### PR TITLE
Jsp1 debug

### PR DESCRIPTION
**Pull request by oysand**

### CODE REVIEW

File: src/isar/mission_planner/local_planner.py [hunk 1]
```diff
@@ -31,6 +39,17 @@ def get_mission(self, mission_id) -> Mission:
...
             mission: Mission = missions[mission_id]["mission"]
             mission.set_unique_mission_id_and_metadata()
+            for mission_step in mission.mission_steps:
+                if isinstance(mission_step, DriveToPose):
+                    mission_step.pose = self.transform.transform_pose(
+                        mission_step.pose, to_=Frame.Robot
+                    )
+                elif isinstance(mission_step, TakeImage) or isinstance(
+                    mission_step, TakeThermalImage
+                ):
```
Comment (tsundvoll) [inline src/isar/mission_planner/local_planner.py:49]:
```suggestion
                elif isinstance(mission_step, (TakeImage, TakeThermalImage):
```

File: src/isar/config/maps/JSP1_weather_deck.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      {
+        "x": 314.621,
+        "y": 108.972,
+        "z": 536.83,
+        "frame": "asset"
+      }
+    ],
+    "frame": "asset"
+  }
+}
```
Comment (tsundvoll) [inline src/isar/config/maps/JSP1_weather_deck.json:49]: This does not need to lie in the version control. I suggest sharing map configs privately and add them to our code personally.

**Review by tsundvoll — CHANGES_REQUESTED** [review]

File: src/isar/config/maps/JSP1_weather_deck.json (hunk 2 above)
Comment (aestene) [reply src/isar/config/maps/JSP1_weather_deck.json:49]: Agreed, however we will approve this one and create an issue to have a more general configuration interchange created.

**Review by aestene — APPROVED** [review]
LGTM

**Pull request merged**